### ISSUE TITLE
Missing translations for location type on check-in screen

### ISSUE THREAD

**Issue by jasperroloff** [corona-warn-app/cwa-app-ios#2564]
## Avoid duplicates
* [x] Bug is not mentioned in the [FAQ](https://www.coronawarn.app/en/faq/)
* [x] Bug is specific for iOS only, for general issues / questions that apply to iOS and Android please raise them in the [documentation repository](https://github.com/corona-warn-app/cwa-documentation)
* [x] Bug is not already reported in another issue

## Technical details

- Device name: iPhone Xs
- iOS version: 14.5
- App version: 2.0.3 (1)

## Describe the bug

When scanning a QR-code to check in at an event or location, the upcoming screen is missing the translation for the "Sonstiges" (unspecified) location type. Instead of a useful title/description, it displays `TraceLocations_Type_Title_Unspecified`.

## Steps to reproduce the issue

1. Go to https://www.coronawarn.app/en/eventregistration/
2. Select "Other" as the event/place type (not "Other event"!)
3. Fill out the rest of the form
4. Click on "Create"
5. Open the CWA iOS app & scan the QR code

## Expected behaviour

Instead of `TraceLocations_Type_Title_Unspecified`, the displayed text should be "Other" or something other descriptive. If there is no suitable translation or, if displaying it makes no sense, the placeholder-text should be invisible.

## Possible Fix

Add translations for this key to `src/xcode/ENA/ENA/Resources/Localization/<Language>.lproj/Localizable.strings`

## Additional context

-

**Comment by Ein-Tim** [comment]
Actually this is an issue with the QR-Code generator, in-app you aren't able to choose "Other".
So the QR-Code generator has to be changed here. The code is correct. 

@dsarkar @svengabr FYI - I suggest a fix ASAP.

**Comment by Ein-Tim** [comment]
I opened https://github.com/corona-warn-app/cwa-website/pull/1169 which fixes this on the QR-Code generator. 
Once the change is live (= the PR is merged), there is no way to create such an event anymore.

It's then up to you and to the developers to decide wether the string should still be defined or not.
